src/sicmutils/env/sci.cljc

```diff
@@ -1,66 +1,111 @@
+;;
+;; Copyright © 2020 Colin Smith.
+;; This work is based on the Scmutils system of MIT/GNU Scheme:
+;; Copyright © 2002 Massachusetts Institute of Technology
+;;
+;; This is free software;  you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation; either version 3 of the License, or (at
+;; your option) any later version.
+;;
+;; This software is distributed in the hope that it will be useful, but
+;; WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+;; General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with this code; if not, see <http://www.gnu.org/licenses/>.
+;;
+
 (ns sicmutils.env.sci
-  (:refer-clojure :exclude [eval])
-  (:require [clojure.set :as set]
-            [sci.core :as sci]
+  (:require [sci.core :as sci]
+            [sicmutils.env]
             [sicmutils.env.sci.macros :as macros]
-            [sicmutils.env :as env]
-            [sicmutils.abstract.function :as af]
-            [sicmutils.calculus.coordinate :as cc]))
-
-(defn ->sci-var [[var-name the-var]]
-  [var-name (cond-> (deref the-var)
-              (-> the-var meta :macro)
```

Comment:
I think this was from an earlier experiment, where we changed the meta, before we realized that we had to rewrite the whole macro. We actually want to filter all macros out.